docker-compose.dev.yml

```diff
@@ -92,3 +92,21 @@ services:
     restart: "no"
   authorization-wrapper:
     restart: "no"
+  deliver-email:
+    image: local-deliver-email-service:latest
```

Comment:
Once the PR https://github.com/redpencilio/deliver-email-service/pull/11 is merged and a new version is published the image here needs to be updated

Reply:
Riad release the new version with node:20 => 0.4.0